README.md

```diff
@@ -56,7 +56,7 @@ Highlights for developers
   * extensible with plugin system
 * interfaced backends for documents parsing and rendering
   * high quality document backend libraries like MuPDF, DjvuLibre and CREngine
-  * interacting with frontend via LuaJIT FFI for best performence
+  * interacting with frontend via LuaJIT FFI for best performance
```

Comment:
Besides the typo I'd also rephrase it slightly. 
```suggestion
  * frontend interaction via LuaJIT FFI for best performance
```

Reply:
@Jason-Cooke Could you please commit (or object to/comment on) my suggestion? :-)

Reply:
@Frenzie Ya sure